apps/accounts/models/automations/backup_account.py

```diff
@@ -47,6 +51,8 @@ class AccountBackupAutomation(PeriodTaskModelMixin, JMSOrgBaseModel):
         max_length=4096, blank=True, null=True, verbose_name=_('Zip encrypt password')
     )
 
+    objects = BaseBackupAutomationManager.from_queryset(models.QuerySet)()
+
     def __str__(self):
         return f'{self.name}({self.org_id})'
 
```

Comment:
It's difficult to assess the code discrepancies without specific examples or additional context about the environment. Please share more information or show the parts of codes where you need analysis or improvement.